app/config.py

```diff
@@ -2,7 +2,7 @@
 from typing import Any
 
 from fastapi.responses import HTMLResponse
-from pydantic import BaseSettings
+from pydantic_settings import BaseSettings
```

Comment:
Thank you so much! Since the `requirements.txt` dependencies are not pinned , it appears that FastAPI is now using pydantic v2, so this is a necessary change. Thank you.